src/createServer.js

```diff
@@ -1,11 +1,20 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
+
+const { userRouter } = require('./routes/user.route.js');
+const { expenseRouter } = require('./routes/expense.route.js');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  app.use(cors());
+
+  app.use('/users', express.json(), userRouter);
+  app.use('/expenses', express.json(), expenseRouter);
+
+  return app;
```

Comment:
you need to initialize empty arrays for users and expenses on each createServer running